### PR TITLE
feat: Expose variable "regex_replace_chars" 

### PR DESCRIPTION
**Pull request by k911**
Motivation: Allow changing label module behavior

### CODE REVIEW

**Comment by terranisu** [comment]
Today I've stumbled upon a similar issue. I tried to create an ECR repository with a name in the following format `group/subgroup/my-awesome-app` (which is an absolutely valid repository name) but didn't succeed.

The downstream [terraform-null-label](https://github.com/cloudposse/terraform-null-label/blob/master/variables.tf#L91-L95) module removes everything except alpha characters, numbers, and dashes, and there are no options to change removal rules.

File: main.tf [hunk 1]
```diff
@@ -5,14 +5,15 @@ locals {
...
 module "label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.14.1"
-  enabled    = var.enabled
-  namespace  = var.namespace
-  stage      = var.stage
-  name       = var.name
-  delimiter  = var.delimiter
-  attributes = var.attributes
-  tags       = var.tags
+  source              = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.14.1"
```
Comment (aknysh) [inline main.tf]:
```suggestion
  source              = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
```

let's update to the latest version

Comment (terranisu) [reply main.tf]: What do you think of merging this one first: https://github.com/cloudposse/terraform-aws-ecr/pull/35 and then rebasing this PR against the master. In that case, this particular PR will stay cleaner introducing only one feature.

Comment (aknysh) [reply main.tf]: agree
testing #35  now and will merge if ok

**Comment by k911** [comment]
@aknysh So, it seems I don't have to change anything here because I've run that command before

**Comment by aknysh** [comment]
@k911 
please rebase

also make sure you run `terraform fmt` and rebuild README (if it's changed or new vars/outputs added):

```
make init
make readme/deps
make readme
``

**Comment by k911** [comment]
@aknysh Done

**Pull request merged**